src/api/roles/shared/chat.py

```
@@ -4,81 +4,103 @@
+
+    if recipient is None or recipient.id is None:
+        raise HTTPException(404, detail="Recipient account not found")
+
+    return recipient
 
 
     
 
-@router.post("/send_message/{chat_id}", response_model=SendMessageResponse)
+@router.post("/messages/{chat_id}", response_model=SendMessageResponse)
 def send_message(chat_id: int, message_text: str, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)